pkg/grpc/api_actor_test.go

```diff
@@ -134,6 +134,9 @@ func TestGetActorState(t *testing.T) {
 			Key:       "key1",
 		}).Return(&actors.StateResponse{
 			Data: data,
+			Metadata: map[string]string{
+				"ttlExpireTime": "2020-10-02T22:00:00Z",
```

Comment:
Sorry to be that person, but why not just "expiration"? Is "ttlExpireTime" used in other systems and we want to mimic?

Reply:
`expiration` could be ambiguous. It is clearer that `ttlExpireTime` refers to and pairs with the `ttl` field.

Reply:
"ttl" is a duration: "1min" or "3 hours". In this case, it means the expiration timestamp, like the date we see stamped at food labels.

Reply:
Yes, `ttl` is what is given in the metadata when saving and is a duration. `ttlExpireTime` is on the get response and is the timestamp of when that `ttl` duration will take effedct.

Reply:
Can we shorten it? `ttlExpires` similar to HTTP `Expires` header but more qualified to `ttl` expiry. Wdyt?

Reply:
It will need to be changed on the contrib side, as [that is where it is coming from](https://github.com/dapr/components-contrib/pull/2863). It is used here in the unit test as a realistic example.